src/containers/Posts/Posts.js

```diff
@@ -2,23 +2,43 @@ import React, {Component} from 'react';
 import Post from "../../components/Post/Post";
 import Navigations from "../../components/Navigations/Navigations";
 import classes from './Posts.module.css'
+import {connect} from "react-redux";
+import * as actions from '../../store/actions/index';
 
 class Posts extends Component {
+    componentDidMount() {
+        this.props.onShowPosts();
+    }
+
     render() {
+        let posts = this.props.posts.map(post =>
+            <Post key={post.title}>{post.description}</Post>
```

Comment:
The title might be duplicated. We can use the genareted key from firebase as key